logic/src/androidMain/kotlin/com/wire/kalium/logic/CoreLogic.kt

```diff
@@ -26,7 +26,7 @@ actual class CoreLogic(
         val dataSourceSet = userScopeStorage[session] ?: run {
             val networkContainer = AuthenticatedNetworkContainer(
                 sessionCredentials = sessionMapper.toSessionCredentials(session),
-                backendConfig = serverConfigMapper.toBackendConfig(serverConfig = session.serverConfig)
+                backendConfig = serverConfigMapper.toBackendConfig(serverConfig = session.serverConfig), isRequestLoggingEnabled = true
```

Comment:
maybe add a TODO so we can remove `isRequestLoggingEnabled = true` later when we have a better solution for logging

Reply:
Oh shit. You found my debugging!

I'll remove this for now.